applications/StructuralMechanicsApplication/custom_constitutive/plasticity_isotropic_kinematic_j2.cpp

```diff
@@ -0,0 +1,525 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Fernando Rastellini
+//  Collaborators:   Riccardo Rossi
+//                   Vicente Mataix Ferrandiz
+//
+
+#include "plasticity_isotropic_kinematic_j2.h"
+#include "structural_mechanics_application_variables.h"
+
+namespace Kratos
+{
+//******************************CONSTRUCTOR*******************************************
+//************************************************************************************
+
+PlasticityIsotropicKinematicJ2::PlasticityIsotropicKinematicJ2()
+    : ConstitutiveLaw()
+{
+}
+
+//********************************COPY CONSTRUCTOR************************************
+//************************************************************************************
+
+PlasticityIsotropicKinematicJ2::PlasticityIsotropicKinematicJ2(const PlasticityIsotropicKinematicJ2 &rOther)
+    : ConstitutiveLaw(rOther)
+{
+}
+
+//********************************CLONE***********************************************
+//************************************************************************************
+
+ConstitutiveLaw::Pointer PlasticityIsotropicKinematicJ2::Clone() const
+{
+    PlasticityIsotropicKinematicJ2::Pointer p_clone(new PlasticityIsotropicKinematicJ2(*this));
```

Comment:
Change this for Kratos::make_shared

https://github.com/KratosMultiphysics/Kratos/blob/698f962c05c5b062bdb0906d276b170daeed1a3d/applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp#L45